src/Abstractions/NexusMods.Abstractions.FileStore/Downloads/DownloadAnalysis.cs

```diff
@@ -10,6 +10,7 @@
 using NexusMods.MnemonicDB.Storage;
 using NexusMods.Paths;
 using ModFileTreeNode = NexusMods.Paths.Trees.KeyedBox<NexusMods.Paths.RelativePath, NexusMods.Abstractions.FileStore.Trees.ModFileTree>;
+using ULongAttribute = NexusMods.Abstractions.MnemonicDB.Attributes.ULongAttribute;
```

Comment:
This attribute exists in two places, we should probably remove our custom one and just use the DB attribute.